CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `commons-io:commons-io` from 2.12.0 to 2.13.0 (#8014, #8013, #8010)
 - Bump `com.diffplug.spotless` from 6.18.0 to 6.19.0 (#8007)
 - Bump `'com.azure:azure-storage-blob` to 12.22.2 from 12.21.1 ([#8043](https://github.com/opensearch-project/OpenSearch/pull/8043))
+- Bump `com.google.guava:guava` from 32.0.0-jre to 32.0.1-jre in /distribution/tools/upgrade-cli ([#8011](https://github.com/opensearch-project/OpenSearch/pull/8011))
```

Comment:
This one is probably not needed?

Reply:
Hmmm, it's the same dependency but in two different paths - `/distribution/tools/upgrade-cli` vs `/distribution/tools/plugin-cli`. 

@reta Do you think we should simplify the changelog entry as follows?

```
Bump com.google.guava:guava from 32.0.0-jre to 32.0.1-jre
```

Reply:
I think we usually follow the pattern `Bump com.google.guava:guava from 32.0.0-jre to 32.0.1-jre (#8011, #8022)`